src/components/MerchantPayout/MerchantPayoutScreen.tsx

```diff
@@ -0,0 +1,153 @@
+import React, { FunctionComponent, useContext, useState } from "react";
+import {
+  View,
+  StyleSheet,
+  KeyboardAvoidingView,
+  ScrollView,
+  Alert
+} from "react-native";
+import { size, color } from "../../common/styles";
+import {
+  withNavigationFocus,
+  NavigationFocusInjectedProps
+} from "react-navigation";
+import { TopBackground } from "../Layout/TopBackground";
+import { useConfigContext } from "../../context/config";
+import { AppHeader } from "../Layout/AppHeader";
+import { ImportantMessageContentContext } from "../../context/importantMessage";
+import { Banner } from "../Layout/Banner";
+import { HelpButton } from "../Layout/Buttons/HelpButton";
+import { FeatureToggler } from "../FeatureToggler/FeatureToggler";
+import { Card } from "../Layout/Card";
+import { Credits } from "../Credits";
+import { HelpModalContext } from "../../context/help";
+import { VoucherInputSection } from "./VoucherInputSection";
+import { DarkButton } from "../Layout/Buttons/DarkButton";
+import { SecondaryButton } from "../Layout/Buttons/SecondaryButton";
+import { Feather } from "@expo/vector-icons";
+import { validateMerchantCode } from "../../utils/validateMerchantCode";
+
+const styles = StyleSheet.create({
+  content: {
+    position: "relative",
+    padding: size(2),
+    paddingVertical: size(8),
+    height: "100%",
+    width: 512,
+    maxWidth: "100%"
+  },
+  headerText: {
+    marginBottom: size(4)
+  },
+  bannerWrapper: {
+    marginBottom: size(1.5)
+  },
+  buttonsWrapper: {
+    marginTop: size(2),
+    flexDirection: "row",
+    alignItems: "center"
+  },
+  submitWrapper: {
+    flex: 1,
+    marginRight: size(1)
+  }
+});
+
+export const MerchantPayoutScreen: FunctionComponent<NavigationFocusInjectedProps> = ({
+  navigation
+}) => {
+  const messageContent = useContext(ImportantMessageContentContext);
+  const { config } = useConfigContext();
+  const showHelpModal = useContext(HelpModalContext);
+  const [vouchers, setVouchers] = useState([{ valid: 5, invalid: 3 }]);
+  const [merchantCode, setMerchantCode] = useState("");
+
+  const submitMerchantCode = (): void => {
+    if (!validateMerchantCode(merchantCode)) {
+      return Alert.alert(
+        "Error",
+        "Invalid Merchant Code",
+        [
+          {
+            text: "Dismiss"
+          }
+        ],
+        {}
+      );
+    }
+    Alert.alert("Valid Merchant Code");
+  };
+
+  const onCancel = (): void => {
+    Alert.alert("Warning", "Cancelling clears all vouchers. Are you sure?", [
+      {
+        text: "Yes",
+        onPress: () => {
+          setVouchers([]);
+          console.log("Yes Pressed");
+        }
+      },
+      {
+        text: "No",
+        onPress: () => console.log("No Pressed")
+      }
+    ]);
```

Comment:
Can refer to this, note the title, position of the buttons and the `destructive` style
https://github.com/rationally-app/mobile-application/blob/f24f219c497e5644d7bd8d47f3bf096d3b0278d7/src/components/CustomerQuota/ItemsSelection/ItemsSelectionCard.tsx#L91